tools/reload_sdk_allowlist.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+SHELL_FOLDER="$(cd "$(dirname $0)" || exit;pwd)"
+
+LOG_ERROR() {
+    content=${1}
+    echo -e "\033[31m[ERROR] ${content}\033[0m"
+}
+
+LOG_INFO() {
+    content=${1}
+    echo -e "\033[32m[INFO] ${content}\033[0m"
+}
+
+cd "${SHELL_FOLDER}/../"
+NODE_FOLDER="$(pwd)"
+fisco_bcos="${NODE_FOLDER}/../fisco-bcos"
+node="$(basename ${NODE_FOLDER})"
```

Comment:
Codacy found an issue: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5822413)